src/main/java/org/craftercms/studio/controller/rest/v2/WorkflowController.java

```diff
@@ -103,6 +107,16 @@ public ResponseBody getItemStates(@RequestParam(name = REQUEST_PARAM_SITEID) Str
         return responseBody;
     }
 
+    private boolean isPathRegexValid(String pathRegex) {
```

Comment:
Is this going to return a 400 (invalid params) to the REST caller or will it silently fail?